dandi/metadata.py

```diff
@@ -361,7 +367,10 @@ def extract_species(metadata):
             value_id = value
             value = None
         else:
-            raise ValueError(f"Cannot interpret species field: {value}")
+            raise ValueError(
+                f"Cannot interpret species field: {value}. Please "
+                "contact help@dandiarchive.org to add your species"
```

Comment:
note: I see emails from Roni testing that this address works, so should be all good.  Adding a trailing `.` to trigger tests rerun etc:
```suggestion
                "contact help@dandiarchive.org to add your species."
```